tests/testthat/test-variable-order.R

```diff
@@ -830,4 +842,76 @@ with_test_authentication({
         expect_identical(urls(ordering(ds)[["Group 2.5"]]),
             urls(variables(ds[c("v4", "v6")])))
     })
+
+    test_that("copyOrder copies across datasets with simple order", {
```

Comment:
These tests have some nice features (test simple and complex orders), but they're too simplistic. A fork has the same variable IDs as its parent, so the translation could falsely appear to succeed if we're doing something sloppy. More importantly, I'd expect that users will be copying orders between datasets that they think are the same but really aren't and have some subtle differences in variables/aliases. So that needs to be handled and tested.